package-lock.json

```diff
@@ -1691,32 +1307,13 @@
       }
     },
     "node_modules/convex-helpers": {
-      "version": "0.1.24",
-      "resolved": "file:packages/convex-helpers",
-      "license": "MIT",
-      "peerDependencies": {
-        "convex": "^1.10.0",
-        "hono": "^4.0.5",
-        "react": "^17.0.2 || ^18.0.0",
-        "zod": "^3.22.4"
-      },
-      "peerDependenciesMeta": {
-        "hono": {
-          "optional": true
-        },
-        "react": {
-          "optional": true
-        },
-        "zod": {
-          "optional": true
-        }
-      }
+      "resolved": "packages/convex-helpers",
+      "link": true
```

Comment:
this change in particular gets annoying - you can restore it by doing `npm i ./packages/convex-helpers` and deleting the convex-helpers package at the bottom of the lockfile